nssp/delphi_nssp/pull.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""Functions for pulling NCHS mortality data API."""
+
+import numpy as np
+import pandas as pd
+from sodapy import Socrata
+
+from .constants import (
+    METRICS,
+    NEWLINE,
+)
+
+
+def construct_typedicts():
```

Comment:
suggestion (optional): since this doesn't take any args, the output will be static. I'd probably move the dictionary def into constants.py instead of regenerating every time on line 59.

Reply:
see: https://github.com/cmu-delphi/covidcast-indicators/pull/1954/files#diff-9eaea874c798327a0c3a60e5a9ac574bd5cb995d8b565fa01f26a2b3412a8743R29

Reply:
updated. https://github.com/cmu-delphi/covidcast-indicators/pull/1952/commits/971968e11bc69c0631e11c534df6b319598f529b